packages/cli/src/projects/command.ts

```diff
@@ -0,0 +1,17 @@
+import yargs from 'yargs';
+import { Opts } from '../options';
+import { ensure, build } from '../util/command-builders';
+import * as o from '../options';
+
+export type ProjectsOptions = Required<Pick<Opts, 'command' | 'projectPath'>>;
+
+const options = [o.projectPath];
+
+const projectsCommand: yargs.CommandModule = {
+  command: 'projects [project-path]',
```

Comment:
No alias to `project`? Although `projects` is more appropriate I think

Reply:
resolved